### ISSUE TITLE
Feature suggestion - support for squadrats.com

### ISSUE THREAD

**Issue by kbrazulewicz** [marcin-gryszkalis/veloviewer-explorer-overlay#3]
Is it technically feasible to support data collected on squadrats.com? Ideally both types of squares should be handled.

**Comment by sebastienaubry** [comment]
Indeed, it would be great to be able to see the visited squares at level 17 (i.e. squadratinhos).

**Comment by momentmal** [comment]
that would indeed be super cool to have a second overlay that covers level 17 tiles instead of level 14 tiles!
